-1 
Tested at: 3c04c6ddf998ec1c14d369ed902395b0bc48991c
 When I ran the RelVals I found an error in the following worklfows: 
 4.53 step2
<pre>runTheMatrix-results/4.53_RunPhoton2012B+RunPhoton2012B+HLTD+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT/step2_RunPhoton2012B+RunPhoton2012B+HLTD+RECODreHLT+HARVESTDreHLT+MINIAODDreHLT.log

----- Begin Fatal Exception 12-May-2015 09:56:08 CEST-----------------------

An exception of category 'FallbackFileOpenError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing input source of type PoolSource

   [2] Calling RootInputFileSequence::initFile()

   [3] Calling StorageFactory::open()

   [4] Calling XrdFile::open()

Exception Message:

Failed to open the file 'root://xrootd-cms.infn.it//store/data/Run2012B/SinglePhoton/RAW/v1/000/194/533/1084D9DA-AEA2-E111-B3AB-001D09F24DA8.root'

   Additional Info:

      [a] XrdCl::File::Open(name='root://eoscms//eos/cms/store/data/Run2012B/SinglePhoton/RAW/v1/000/194/533/1084D9DA-AEA2-E111-B3AB-001D09F24DA8.root?svcClass=default', flags=0x10, permissions=0660) => error '[ERROR] Operation expired' (errno=0, code=206). No additional data servers were found.

      [b] Last URL tried: root://eoscms:1094//eos/cms/store/data/Run2012B/SinglePhoton/RAW/v1/000/194/533/1084D9DA-AEA2-E111-B3AB-001D09F24DA8.root?svcClass=default&tried=

      [c] Problematic data server: eoscms:1094

      [d] Disabled source: eoscms:1094

      [e] Input file root://eoscms//eos/cms/store/data/Run2012B/SinglePhoton/RAW/v1/000/194/533/1084D9DA-AEA2-E111-B3AB-001D09F24DA8.root?svcClass=default could not be opened.

Fallback Input file root://xrootd-cms.infn.it//store/data/Run2012B/SinglePhoton/RAW/v1/000/194/533/1084D9DA-AEA2-E111-B3AB-001D09F24DA8.root also could not be opened.

Original exception info is above; fallback exception info is below.

      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/data/Run2012B/SinglePhoton/RAW/v1/000/194/533/1084D9DA-AEA2-E111-B3AB-001D09F24DA8.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.]:1094 Mw[::134.158.132.31]:1094

' (errno=3011, code=400). No additional data servers were found.

      [g] Last URL tried: root://cms-xrd-global.cern.ch:1094//store/data/Run2012B/SinglePhoton/RAW/v1/000/194/533/1084D9DA-AEA2-E111-B3AB-001D09F24DA8.root?tried=+1213llrxrd-redir.in2p3.fr

      [h] Problematic data server: cms-xrd-global.cern.ch:1094

      [i] Disabled source: cms-xrd-global.cern.ch:1094

----- End Fatal Exception -------------------------------------------------

</pre>
4.22 step2
<pre>runTheMatrix-results/4.22_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC/step2_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC.log

----- Begin Fatal Exception 12-May-2015 09:56:41 CEST-----------------------

An exception of category 'FallbackFileOpenError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing input source of type PoolSource

   [2] Calling RootInputFileSequence::initFile()

   [3] Calling StorageFactory::open()

   [4] Calling XrdFile::open()

Exception Message:

Failed to open the file 'root://xrootd-cms.infn.it//store/data/Run2011A/Cosmics/RAW/v1/000/160/960/049F6443-8E53-E011-A943-003048F117EA.root'

   Additional Info:

      [a] XrdCl::File::Open(name='root://eoscms//eos/cms/store/data/Run2011A/Cosmics/RAW/v1/000/160/960/049F6443-8E53-E011-A943-003048F117EA.root?svcClass=default', flags=0x10, permissions=0660) => error '[ERROR] Operation expired' (errno=0, code=206). No additional data servers were found.

      [b] Last URL tried: root://eoscms:1094//eos/cms/store/data/Run2011A/Cosmics/RAW/v1/000/160/960/049F6443-8E53-E011-A943-003048F117EA.root?svcClass=default&tried=

      [c] Problematic data server: eoscms:1094

      [d] Disabled source: eoscms:1094

      [e] Input file root://eoscms//eos/cms/store/data/Run2011A/Cosmics/RAW/v1/000/160/960/049F6443-8E53-E011-A943-003048F117EA.root?svcClass=default could not be opened.

Fallback Input file root://xrootd-cms.infn.it//store/data/Run2011A/Cosmics/RAW/v1/000/160/960/049F6443-8E53-E011-A943-003048F117EA.root also could not be opened.

Original exception info is above; fallback exception info is below.

      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/data/Run2011A/Cosmics/RAW/v1/000/160/960/049F6443-8E53-E011-A943-003048F117EA.root', flags=0x10, permissions=0660) => error '[FATAL] Redirect limit has been reached' (errno=0, code=306). No additional data servers were found.

      [g] Last URL tried: root://cmsxrootd1.fnal.gov:1094//store/data/Run2011A/Cosmics/RAW/v1/000/160/960/049F6443-8E53-E011-A943-003048F117EA.root?tried=+1213cmsxrootd1.fnal.gov1213xrootd.unl.edu

      [h] Problematic data server: cmsxrootd1.fnal.gov:1094

      [i] Disabled source: cmsxrootd1.fnal.gov:1094

----- End Fatal Exception -------------------------------------------------

</pre>
140.53 step2
<pre>runTheMatrix-results/140.53_RunHI2011+RunHI2011+RECOHID11+HARVESTDHI/step2_RunHI2011+RunHI2011+RECOHID11+HARVESTDHI.log

----- Begin Fatal Exception 12-May-2015 09:56:56 CEST-----------------------

An exception of category 'FallbackFileOpenError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing input source of type PoolSource

   [2] Calling RootInputFileSequence::initFile()

   [3] Calling StorageFactory::open()

   [4] Calling XrdFile::open()

Exception Message:

Failed to open the file 'root://xrootd-cms.infn.it//store/hidata/HIRun2011/HIMinBiasUPC/RAW/v1/000/182/124/0011F55A-8F13-E111-A987-003048F118C6.root'

   Additional Info:

      [a] XrdCl::File::Open(name='root://eoscms//eos/cms/store/hidata/HIRun2011/HIMinBiasUPC/RAW/v1/000/182/124/0011F55A-8F13-E111-A987-003048F118C6.root?svcClass=default', flags=0x10, permissions=0660) => error '[ERROR] Operation expired' (errno=0, code=206). No additional data servers were found.

      [b] Last URL tried: root://eoscms:1094//eos/cms/store/hidata/HIRun2011/HIMinBiasUPC/RAW/v1/000/182/124/0011F55A-8F13-E111-A987-003048F118C6.root?svcClass=default&tried=

      [c] Problematic data server: eoscms:1094

      [d] Disabled source: eoscms:1094

      [e] Input file root://eoscms//eos/cms/store/hidata/HIRun2011/HIMinBiasUPC/RAW/v1/000/182/124/0011F55A-8F13-E111-A987-003048F118C6.root?svcClass=default could not be opened.

Fallback Input file root://xrootd-cms.infn.it//store/hidata/HIRun2011/HIMinBiasUPC/RAW/v1/000/182/124/0011F55A-8F13-E111-A987-003048F118C6.root also could not be opened.

Original exception info is above; fallback exception info is below.

      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/hidata/HIRun2011/HIMinBiasUPC/RAW/v1/000/182/124/0011F55A-8F13-E111-A987-003048F118C6.root', flags=0x10, permissions=0660) => error '[FATAL] Redirect limit has been reached' (errno=0, code=306). No additional data servers were found.

      [g] Last URL tried: root://cmsxrootd1.fnal.gov:1094//store/hidata/HIRun2011/HIMinBiasUPC/RAW/v1/000/182/124/0011F55A-8F13-E111-A987-003048F118C6.root?tried=+1213cmsxrootd1.fnal.gov1213xrootd.unl.edu

      [h] Problematic data server: cmsxrootd1.fnal.gov:1094

      [i] Disabled source: cmsxrootd1.fnal.gov:1094

----- End Fatal Exception -------------------------------------------------

</pre>
1001.0 step2
<pre>runTheMatrix-results/1001.0_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD/step2_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD.log

----- Begin Fatal Exception 12-May-2015 10:02:26 CEST-----------------------

An exception of category 'FallbackFileOpenError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing input source of type PoolSource

   [2] Calling RootInputFileSequence::initFile()

   [3] Calling StorageFactory::open()

   [4] Calling XrdFile::open()

Exception Message:

Failed to open the file 'root://xrootd-cms.infn.it//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root'

   Additional Info:

      [a] XrdCl::File::Open(name='root://eoscms//eos/cms/store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?svcClass=default', flags=0x10, permissions=0660) => error '[ERROR] Operation expired' (errno=0, code=206). No additional data servers were found.

      [b] Last URL tried: root://eoscms:1094//eos/cms/store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?svcClass=default&tried=

      [c] Problematic data server: eoscms:1094

      [d] Disabled source: eoscms:1094

      [e] Input file root://eoscms//eos/cms/store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?svcClass=default could not be opened.

Fallback Input file root://xrootd-cms.infn.it//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root also could not be opened.

Original exception info is above; fallback exception info is below.

      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to open /store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [g] Last URL tried: root://node12.datagrid.cea.fr:11001//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?tried=

      [h] Problematic data server: node12.datagrid.cea.fr:11001

      [i] Disabled source: node12.datagrid.cea.fr:11001

----- End Fatal Exception -------------------------------------------------

</pre>
1003.0 step2
<pre>runTheMatrix-results/1003.0_RunMinBias2012A+RunMinBias2012A+RECODDQM+HARVESTDDQM/step2_RunMinBias2012A+RunMinBias2012A+RECODDQM+HARVESTDDQM.log

----- Begin Fatal Exception 12-May-2015 10:02:34 CEST-----------------------

An exception of category 'FallbackFileOpenError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing input source of type PoolSource

   [2] Calling RootInputFileSequence::initFile()

   [3] Calling StorageFactory::open()

   [4] Calling XrdFile::open()

Exception Message:

Failed to open the file 'root://xrootd-cms.infn.it//store/data/Run2012A/MinimumBias/RAW/v1/000/191/226/02F7EB61-0986-E111-A66F-5404A63886B4.root'

   Additional Info:

      [a] XrdCl::File::Open(name='root://eoscms//eos/cms/store/data/Run2012A/MinimumBias/RAW/v1/000/191/226/02F7EB61-0986-E111-A66F-5404A63886B4.root?svcClass=default', flags=0x10, permissions=0660) => error '[ERROR] Operation expired' (errno=0, code=206). No additional data servers were found.

      [b] Last URL tried: root://eoscms:1094//eos/cms/store/data/Run2012A/MinimumBias/RAW/v1/000/191/226/02F7EB61-0986-E111-A66F-5404A63886B4.root?svcClass=default&tried=

      [c] Problematic data server: eoscms:1094

      [d] Disabled source: eoscms:1094

      [e] Input file root://eoscms//eos/cms/store/data/Run2012A/MinimumBias/RAW/v1/000/191/226/02F7EB61-0986-E111-A66F-5404A63886B4.root?svcClass=default could not be opened.

Fallback Input file root://xrootd-cms.infn.it//store/data/Run2012A/MinimumBias/RAW/v1/000/191/226/02F7EB61-0986-E111-A66F-5404A63886B4.root also could not be opened.

Original exception info is above; fallback exception info is below.

      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/data/Run2012A/MinimumBias/RAW/v1/000/191/226/02F7EB61-0986-E111-A66F-5404A63886B4.root', flags=0x10, permissions=0660) => error '[FATAL] Redirect limit has been reached' (errno=0, code=306). No additional data servers were found.

      [g] Last URL tried: root://cmsxrootd1.fnal.gov:1094//store/data/Run2012A/MinimumBias/RAW/v1/000/191/226/02F7EB61-0986-E111-A66F-5404A63886B4.root?tried=+1213cmsxrootd1.fnal.gov1213xrootd.unl.edu

      [h] Problematic data server: cmsxrootd1.fnal.gov:1094

      [i] Disabled source: cmsxrootd1.fnal.gov:1094

----- End Fatal Exception -------------------------------------------------

</pre>
1000.0 step2
<pre>runTheMatrix-results/1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT/step2_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT.log

----- Begin Fatal Exception 12-May-2015 10:02:42 CEST-----------------------

An exception of category 'FallbackFileOpenError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing input source of type PoolSource

   [2] Calling RootInputFileSequence::initFile()

   [3] Calling StorageFactory::open()

   [4] Calling XrdFile::open()

Exception Message:

Failed to open the file 'root://xrootd-cms.infn.it//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root'

   Additional Info:

      [a] XrdCl::File::Open(name='root://eoscms//eos/cms/store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?svcClass=default', flags=0x10, permissions=0660) => error '[ERROR] Operation expired' (errno=0, code=206). No additional data servers were found.

      [b] Last URL tried: root://eoscms:1094//eos/cms/store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?svcClass=default&tried=

      [c] Problematic data server: eoscms:1094

      [d] Disabled source: eoscms:1094

      [e] Input file root://eoscms//eos/cms/store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?svcClass=default could not be opened.

Fallback Input file root://xrootd-cms.infn.it//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root also could not be opened.

Original exception info is above; fallback exception info is below.

      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3010] Unable to open /store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root; Permission denied

' (errno=3010, code=400). No additional data servers were found.

      [g] Last URL tried: root://node12.datagrid.cea.fr:11001//store/data/Run2011A/MinimumBias/RAW/v1/000/165/121/0699429A-B37F-E011-A57A-0019B9F72D71.root?tried=

      [h] Problematic data server: node12.datagrid.cea.fr:11001

      [i] Disabled source: node12.datagrid.cea.fr:11001

----- End Fatal Exception -------------------------------------------------

</pre>

 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-8981/35/summary.html

--- a/Configuration/PyReleaseValidation/python/relval_standard.py
+++ b/Configuration/PyReleaseValidation/python/relval_standard.py
@@ -240,7 +240,7 @@ workflows[1354] = ['WpToENu_M-2000_13TeV', ['WpToENu_M-2000_13TeV','DIGIUP15','R
 
 ### HI test ###
 
-###  Run I
+###  Run I cond.
 workflows[140] = ['',['HydjetQ_MinBias_2760GeV','DIGIHI2011','RECOHI2011','HARVESTHI2011']]
 
 workflows[140.1] = ['',['QCD_Pt_80_120_13_HI','DIGIHI','RECOHI','HARVESTHI']]
@@ -251,7 +251,7 @@ workflows[140.4] = ['',['ZEEMM_13_HI','DIGIHI','RECOHI','HARVESTHI']]
 # legacy: 141 => B0; 143 => B3 ; 144 => B5 
 #workflows[142] = ['',['HydjetQ_B8_2760GeV','DIGIHI','RECOHI','HARVESTHI','MINIAODMCUP15HI']]
 
-### Run II
+### Run II cond.
 workflows[145] = ['',['HydjetQ_MinBias_5020GeV','DIGIHI','RECOHI','HARVESTHI']]
 
 ### pPb test ###

--- a/Configuration/PyReleaseValidation/python/relval_standard.py
+++ b/Configuration/PyReleaseValidation/python/relval_standard.py
@@ -242,7 +242,6 @@ workflows[1354] = ['WpToENu_M-2000_13TeV', ['WpToENu_M-2000_13TeV','DIGIUP15','R
 
 ###  Run I cond.
 workflows[140] = ['',['HydjetQ_MinBias_2760GeV','DIGIHI2011','RECOHI2011','HARVESTHI2011']]
-
 workflows[140.1] = ['',['QCD_Pt_80_120_13_HI','DIGIHI','RECOHI','HARVESTHI']]
 workflows[140.2] = ['',['PhotonJets_Pt_10_13_HI','DIGIHI','RECOHI','HARVESTHI']]
 workflows[140.3] = ['',['ZMM_13_HI','DIGIHI','RECOHI','HARVESTHI']]

--- a/Configuration/PyReleaseValidation/python/relval_standard.py
+++ b/Configuration/PyReleaseValidation/python/relval_standard.py
@@ -239,6 +239,7 @@ workflows[1353] = ['RSGravitonToGaGa_13TeV', ['RSGravitonToGaGa_13TeV','DIGIUP15
 workflows[1354] = ['WpToENu_M-2000_13TeV', ['WpToENu_M-2000_13TeV','DIGIUP15','RECOUP15','HARVESTUP15','MINIAODMCUP15']]
 
 ### HI test ###
+
 ###  Run I
 workflows[140] = ['',['HydjetQ_MinBias_2760GeV','DIGIHI2011','RECOHI2011','HARVESTHI2011']]
 
@@ -249,7 +250,6 @@ workflows[140.4] = ['',['ZEEMM_13_HI','DIGIHI','RECOHI','HARVESTHI']]
 
 # legacy: 141 => B0; 143 => B3 ; 144 => B5 
 #workflows[142] = ['',['HydjetQ_B8_2760GeV','DIGIHI','RECOHI','HARVESTHI','MINIAODMCUP15HI']]
-
 
 ### Run II
 workflows[145] = ['',['HydjetQ_MinBias_5020GeV','DIGIHI','RECOHI','HARVESTHI']]

